### DISCUSSION TITLE
v3用raidrive挂载提示Protocol version

### DISCUSSION THREAD

**Discussion by CRYCATCRY** [AlistGo/alist#1919] [Q&A]
想起来控制面板网址好像是http没有加密，原因是这个

**Comment by Shao-zp** [comment]
我也是这么提示的，应该怎么解决啊

**Reply by GengQW** [reply]
![image](https://user-images.githubusercontent.com/102752573/196048054-086b91ee-a575-4d3d-9a51-53686a00e85f.png)

**Reply by GengQW** [reply]
取消勾选

**Reply by sanroot3** [reply]
> 取消勾选

感谢大佬解惑~

**Reply by jun-141013** [reply]
> 取消勾选

为什么我取消勾选了会报403

**Reply by OMG55555** [reply]
那就是你地址填错了

**Reply by SUKdd** [reply]
牛逼大佬
